tinydrive/tinydrive-api/plugin.md

```diff
@@ -77,7 +77,7 @@ This setting enables constraining the width/height of uploaded images. When this
 ```js
 tinydrive.pick({
   max_image_dimension: 1024
-}).then((result) => {
+}).then( function (result) {
```

Comment:
There probably shouldn't be a space before `function` here.

Reply:
Fixed in https://github.com/tinymce/tinymce-docs/pull/1645/commits/50d665fdbcdb9afd3395dccf0ff6eba409f4895d